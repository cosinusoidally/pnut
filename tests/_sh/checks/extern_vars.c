// expect_comp_failure
extern int some_var;

void main() {

}
