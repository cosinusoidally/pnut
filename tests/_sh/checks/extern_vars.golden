tests/_sh/checks/extern_vars.c:3:0  Extern storage class specifier not supported
